### PR TITLE
Revert "net: lib: nrf_cloud: Update CoAP to support QZSS"

### PR DESCRIPTION
**Pull request by plskeggs**
This reverts commit 43dcee5b87298781746df22bb6fcae76cd3c35d8.

The agnss resource is not yet deployed to the
production server.

### CODE REVIEW

**Review by trantanen — APPROVED** [review]

**Comment by plskeggs** [comment]
Note to merge person: I am a code owner and I give permission for this PR to be merged without approvals from the other code owners, who are members of my team.  We need to expedite this due to the original commit breaking another team's CI test.

**Review by jayteemo — APPROVED** [review]

**Review by tokangas — APPROVED** [review]

**Pull request merged**
